src/modules/schema.js

```diff
@@ -137,10 +137,10 @@ export default class Schema {
    * Prepares a transaction to write this schema object to the dock chain using the blob module
    * @param {object} dock - The dock API
    * @param {object} pair - The keypair to sign with
-   * @return {object} The extrinsic to sign and send.
+   * @return {Promise<object>} The extrinsic to sign and send.
    */
-  writeToChain(dock, pair) {
-    return dock.blob.new(this.toBlob(), pair);
+  async writeToChain(dock, pair) {
+    return await dock.blob.new(this.toBlob(), pair, false);
```

Comment:
You can omit the false.

Reply:
can't recall 100% why for blob module we are passing false here, maybe something fausto encountered as he developed it. i'll see what happens if we remove this

Reply:
seems to be fine when removed